tiny_dnn/layers/layer.h

```diff
@@ -708,6 +687,53 @@ class layer : public node {
     }
   }
 
+  ////////////////////////////////////////////////////////////////////////////
+  // getters setters for parameters
```

Comment:
This is not a function docstring, but enclosing a set of functions, similar to this line. https://github.com/tiny-dnn/tiny-dnn/pull/754/files/d7e647d109f2d40af00f92bbfa0f0977048ce9c4#diff-a1617ac70e8f704e6996e605b9f4b889R374

Shall I modify these as well ?